src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -23,7 +23,8 @@
     <UseFullSemVerForNuGet>false</UseFullSemVerForNuGet>
     <Title>CommunityToolkit.Maui</Title>
     <Description>The .NET MAUI Community Toolkit is a collection of Animations, Behaviors, Converters, and Effects for development with .NET MAUI. It simplifies and demonstrates common developer tasks building iOS, Android, macOS and Windows apps with .NET MAUI.</Description>
-    <PackageIcon>icon.png</PackageIcon>
+    <!-- TODO: Get a new amazing icon! -->
+    <!--<PackageIcon>icon.png</PackageIcon>-->
```

Comment:
WE NEED AN ICON!

Reply:
We have our general Toolkit icon from the WCT repo, feel free to just copy that 😉 https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/build/nuget.png

It's the B/W Toolbox image that we use as the 'Toolkit' icon and that I made the organization icon.

Reply:
Added that for now!